src/components/LaboratoryChrome.js

```diff
@@ -28,6 +34,33 @@ function LaboratoryChrome(props) {
   }
 
   return <div>
+    <style dangerouslySetInnerHTML={{__html: `
```

Comment:
Is it possible to put these in a css file and just import that?

Reply:
There isn't a mechanism for hosting static assets that I could find besides importing them, and I'm not clear how to do so from a non-JS file.

Reply:
Does this transform it into base64? I was able to import SVGs into my CSS files as base64 dataurls using css-loader, svg-loader, and praying to the gods of webkit: https://github.com/stellar/sep6-demo-client/blob/master/webpack.config.js

what you have now certainly doesn't bother me though.

Reply:
https://webpack.js.org/loaders/css-loader/
`The css-loader interprets @import and url() like import/require() and will resolve them.`

Reply:
Ooo good tip. Cheers, I've been deep in styled-components for a long time.